### DISCUSSION TITLE
Sometimes my printer stows the probe while building a mesh.

### DISCUSSION THREAD

**Discussion by Heros-Tempus** [mriscoc/Ender3V2S1#1017] [Q&A]
I've been using the August pre-release firmware on my ender 3 s1. I have absolutely no idea why this happens, but occasionally while constructing a mesh the probe tip will stow before it comes down on the test point. Since the probe tip is stowed, the head winds up moving down further than expected, then the printer panics and it is killed before the head crashes into the bed. Usually I am able to just turn it off then on then start building the mesh again without issue, but it would be nice to know what is causing this.

**Comment by mriscoc** [comment]
CRTouch? How is the HS mode?
https://github.com/mriscoc/Ender3V2S1/wiki/3D-BLTouch#notes-about-crtouch

**Reply by Heros-Tempus** [reply]
HS mode was on. I've turned it off. I'm not certain if that fixed the problem because I don't know how that mode effects the machine and I've only built one mesh since turning it off. Since I don't know how that mode works, successfully building one mesh could be attributed to luck.
